wear/src/main/java/io/homeassistant/companion/android/home/views/MainView.kt

```diff
@@ -321,7 +318,7 @@ fun MainView(
                                             }.sorted()
                                         ) { true }
                                     },
-                                    colors = ChipDefaults.secondaryChipColors()
+                                    colors = ButtonDefaults.filledTonalButtonColors()
```

Comment:
These buttons have no visible background? Wouldn't it make more sense to have the same buttons as the settings items, like we did before? Same applies for sensor managers.
![image](https://github.com/home-assistant/android/assets/8148535/06323916-c2fc-4256-869b-2a461e6153f2)

Reply:
ah yes will fix in next commit :)